lib/Test2/Compare/Object.pm

```diff
@@ -74,6 +75,18 @@ sub add_call {
     push @{$self->{+CALLS}} => [$meth, $check, $name, $context || 'scalar'];
 }
 
+sub add_this_isa {
+    my $self = shift;
+    my ($class_name) = @_;
```

Comment:
Should we accept extra arguments to allow overriding arguments of `Test2::Compare::Isa->new`?

```perl
my ($class_name, @args) = @_;
...
Test2::Compare::isa->new(..., @args);
```

```perl
object {
    this_isa 'Foo::Bar', file => ..., line => ...;
}
```

Reply:
not necessary.